css/css-backgrounds/border-image-slice-fill-003.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Backgrounds Test: 'border-image-slice: 0' with 'fill' keyword (edge case)</title>
+
+  <!--
+
+  Created: February 24th 2023
+
+  Last modified: February 28th 2023
+
+  -->
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="http://www.w3.org/TR/css3-background/#border-image-slice">
+  <link rel="match" href="reference/ref-filled-green-100px-square.xht">
+
+  <meta content="" name="flags">
+  <meta content="This test checks the rendering process of selecting the middle part of the border-image to be preserved. Since 'border-image-slice' is '0', then the whole border-image is its own middle part. In which case, the horizontal center and the vertical center of the border-image must fill the middle area." name="assert">
+
+  <style>
+  div
+    {
+      border: red none 0px;
+      border-image-repeat: repeat;
+      border-image-slice: 0 fill;
```

Comment:
```suggestion
      border: red none 100px;
      border-image-repeat: repeat;
      border-image-slice: 100 fill;
```

This will not change in passing UAs, but if the UA fails to make the border 0px (as required by `none`) or it applies any part of the outer border slices at more than 0px, we will see red.